From da7f2247c9dbe631c0e221725dbda1517bb471c3 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Thu, 25 Jan 2024 17:32:25 +0100
Subject: [PATCH 473/841] refactor(st-drivers): remove useless structure fields
 in fixed regul

Cleanup useless structure fields in order to
reduce code size.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git b905c81930b4b49ce67d544b4ecfb14767103e7d refactor(st-drivers): remove useless structure fields in fixed regul]
Change-Id: Ifb1df2d347c3d892d6633b339fa0bae648c87537
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/354522
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 drivers/st/regulator/regulator_fixed.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/st/regulator/regulator_fixed.c b/drivers/st/regulator/regulator_fixed.c
index 6c9d3b17e..b98e449ac 100644
--- a/drivers/st/regulator/regulator_fixed.c
+++ b/drivers/st/regulator/regulator_fixed.c
@@ -21,7 +21,6 @@
 
 struct fixed_data {
 	char name[FIXED_NAME_LEN];
-	uint16_t volt;
 	struct regul_description desc;
 };
 
-- 
2.47.2

